app/views/shared/_upcoming_events.html.erb

```diff
@@ -0,0 +1,74 @@
+<div class="container">
```

Comment:
I used it from Ewoud, who made something similar for the "Choose your course" section, and adjusted it. Not sure why it shows the weird indentation. It looks fine in atom:

![screenshot from 2017-03-30 18-36-08](https://cloud.githubusercontent.com/assets/24799406/24515412/d017fa0c-1577-11e7-9208-7e6b7be185e9.png)